lib/capsize/template/base.rb

```diff
@@ -4,7 +4,7 @@ module Base
       CONFIG = {
         :application => 'your_app_name',
         :deploy_to => '/path/to/deployment_base',
-        :scm => 'git',
+        :scm => ':git',
```

Comment:
Gotcha — this is really defining the default values for various content placeholders, and `:symbol.to_s` won't give you the correct answer.